iris-mpc-bins/bin/iris-mpc-anon-stats-server/main.rs

```diff
@@ -250,37 +299,92 @@ impl AnonStatsProcessor {
                     "Completed 1D anon stats job"
                 );
 
-                self.publish_1d_stats(&stats).await?;
+                self.publish_1d_stats(operation, &stats).await?;
                 self.store.mark_anon_stats_processed_1d_lifted(&ids).await?;
-                self.sync_failures.remove(&origin);
+                self.sync_failures.remove(&(origin, operation));
                 Ok(())
             }
-            JobKind::Gpu2D => unreachable!(),
+            JobKind::Gpu1DReauth => {
```

Comment:
I think for now this could also be merged with the gpu1d case, IDK if there needs to be different handling in the future but ATM I do not think so, right?